django/views/debug.py

```diff
@@ -294,7 +298,7 @@ def get_traceback_data(self):
             'settings': get_safe_settings(),
             'sys_executable': sys.executable,
             'sys_version_info': '%d.%d.%d' % sys.version_info[0:3],
-            'server_time': timezone.now(),
+            'server_time': localtime(datetime.utcnow().replace(tzinfo=utc)),
```

Comment:
Yes and no, my code now ensures that the datetime is local and aware,
timezone.now returned a naive one, hence the error later on. My call to
localtime is fine since I am working on an aware utc timestamp.

On Mon, Nov 2, 2015, 10:05 Aymeric Augustin <notifications@github.com>
wrote:

> In django/views/debug.py
> <https://github.com/django/django/pull/5529#discussion_r43607653>:
>
> > @@ -294,7 +298,7 @@ def get_traceback_data(self):
> >              'settings': get_safe_settings(),
> >              'sys_executable': sys.executable,
> >              'sys_version_info': '%d.%d.%d' % sys.version_info[0:3],
> > -            'server_time': timezone.now(),
> > +            'server_time': localtime(datetime.utcnow().replace(tzinfo=utc)),
>
> I meant -- isn't the call to localtime going to raise the same exception
> as the template engine? Because the template engine calls this function or
> an equivalent one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/django/django/pull/5529/files#r43607653>.
>